scripts/restart_ecs

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+#
+# Restart ECS service
+#
+
+cluster_name="${APP_ENV}-easi-app"
+service_name="easi-app"
+( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment ) || exit
```

Comment:
```suggestion
( set -x ; aws ecs update-service --cluster "$cluster_name" --service "$service_name" --force-new-deployment )
```
A shell script will exit with the exit status of the last command, so this is not needed anymore here.

Reply:
Good catch! Updated.